app/helpers/admin/analytics_helper.rb

```diff
@@ -8,5 +8,22 @@ def track_analytics_data(category, action, label)
         "track-label" => ActionController::Base.helpers.strip_tags(label),
       }
     end
+
+    def track_analytics_data_on_load(title)
+      {
+        event_name: "page_view",
+        page_view: {
+          government_department_name: "Government Digital Service",
+          access_level: access_based_roles(current_user&.permissions),
+          publishing_app: "Whitehall",
+          experience: current_user&.created_at,
```

Comment:
This experience field bothers me. Perhaps we can rename the key so it actually matches what we are sending + prettify the date?